### PR TITLE
Remove fractal registry from fractal contracts

### PR DESCRIPTION
**Pull request by adamgall**
Please review and merge https://github.com/decentdao/fractal-interface/pull/1652 first.

Remove fractalRegistryContract from FractalContracts interface and resolve all downstream issues

### CODE REVIEW

File: src/hooks/utils/useMasterCopy.ts [hunk 1]
```diff
@@ -73,6 +73,7 @@ export function useMasterCopy() {
       let masterCopyAddress: Address = zeroAddress;
       let error;
       if (baseContracts) {
+        // TODO after removing this moduleproxyfactorycontract from basecontracts, kill getEventRPC
```
Comment (Da-Colon) [inline src/hooks/utils/useMasterCopy.ts:76]: 🔥

Comment (adamgall) [reply src/hooks/utils/useMasterCopy.ts:76]: It's been killed in a future PR!

File: src/hooks/DAO/useDAOName.ts [hunk 2]
```diff
@@ -1,38 +1,27 @@
...
 import { createAccountSubstring } from '../utils/useDisplayName';
 
-/**
- * Gets the 'display name' for a Fractal DAO, in the following order of preference:
- *
- * 1. Primary ENS Name (reverse record)
- * 2. Fractal name registry name
- * 3. Truncated Eth address in the form 0xbFC4...7551
- */
-export default function useDAOName({
```
Comment (DarksightKellar) [inline src/hooks/DAO/useDAOName.ts:18]: Is this doc no longer accurate?

Comment (adamgall) [reply src/hooks/DAO/useDAOName.ts:18]: Correct

**Pull request merged**